biothings/hub/__init__.py

```diff
@@ -29,6 +29,7 @@
 def _config_for_app(config_mod=None):
 
     if not config_mod:
+        sys.path.insert(0, ".")  # import config.py in current folder
```

Comment:
probably should check if the current folder is already in the `sys.path` to avoid adding it repeatedly. @remoteeng00